app/static/js/instructorTable.js

```diff
@@ -0,0 +1,51 @@
+// Instructor manipulation functions
+// -------------------------------------
+
+export function getRowUsername(element) {
+    return $(element).closest("tr").data("username")
+}
+
+export function createNewRow(selectedInstructor) {
+  let instructor = (selectedInstructor["firstName"]+" "+selectedInstructor["lastName"]+" ("+selectedInstructor["email"]+")");
+  let username = selectedInstructor["username"];
+  let phone = selectedInstructor["phoneNumber"];
+  let tableBody = $("#instructorTable").find("tbody");
+
+  if(tableBody.prop('outerHTML').includes(instructor)){
+    msgFlash("Instructor is already added.", "danger");
```

Comment:
For your modal, when I edit it and add instructors to the instructor's table, if I try to add an instructor that was already added to the table, it doesn't let me, which is good. However, the flash message "Instructor is already added." is displayed behind the modal, so I actually cannot see it, which is a bad user experience. I see only the corners of the flash message, and the rest is blurred because we are displayed with the modal. It would be nice to show the flash message on top of modal, so that the user can actually see it

Reply:
Fixed